pom.xml

```diff
@@ -2307,7 +2307,7 @@
       <dependency>
         <groupId>com.fasterxml.jackson.core</groupId>
         <artifactId>jackson-databind</artifactId>
-        <version>${version.com.fasterxml.jackson.databind}</version>
+        <version>${version.com.fasterxml.jackson}</version>
```

Comment:
this will need to be kept as databind is not synced on releases with the whole com.fasterxml.jackson groupId